tools/configen/configen/config.py

```diff
@@ -1,4 +1,6 @@
-# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+# Copyright (c) 2020, Facebook, Inc. and its affiliates. All Rights Reserved
+# SPDX-License-Identifier: MIT
+
```

Comment:
is it about year? or adding SPDX identifier?

Not sure what is your policy, but according to the MIT license 
https://mit-license.org/
you should "permission notice shall be included in all copies or substantial portions of the Software" - so I understand that: header in every file. SPDX initiative "refines" that header to one liner...

Reply:
The current header follows the requirements from Facebook, and is also automatically enforced using various tools (some are external to this repo). I don't think we should change it without an explicit request from Facebook.

Year: This is not a common practice for Facebook projects, I think it's intentionally omitted (to avoid having to fix it every year i guess).

SPDX: This is the first time I run into it, again - not a common practice for Facebook projects.
I would rather keep the headers consistent in each project. Since Hydra has a lot of source files don't think it's worth introducing it.

"The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software." is fullfilled because Hydra comes with the license file.
My understanding is that this refers to other people copying significant parts of the code from Hydra. If they do that need to somehow acknowledge the license.